wire-codegen-sample/src/main/java/com/squareup/wire/sample/ServiceGenerator.java

```diff
@@ -45,19 +45,19 @@ public TypeSpec api(Service service) {
     }
 
     for (Rpc rpc : service.rpcs()) {
-      ProtoType requestType = rpc.requestType();
+      ProtoType requestType = rpc.getRequestType();
```

Comment:
A bunch of these accessors should disappear in follow-up when I convert these types to Kotlin.